chart/templates/replicated-role.yaml

```diff
@@ -11,5 +11,16 @@ rules:
   resources:
   - '*'
   verbs:
-  - '*'
+  - 'get'
+  - 'list'
+  - 'watch'
+- apiGroups:
+  - ''
+  resources:
+  - 'configmaps'
```

Comment:
will we still update the configmap, or can we take this out too?

Reply:
good point, i'm fine with leaving it for now. we have "update" permission for a secret named "replicated", don't see the harm in having that as well for a configmap named "replicated"